File: backend/kanban/kanban_user.go
package kanban

import "main/repository/models"

type KanbanUser struct {
	Id        uint            `json:"id"`
	Name      string          `json:"name"`
	Username  string          `json:"username"`
	AvatarUrl string          `json:"avatarUrl"`
	Issues    []Issue         `json:"issues"`
	Teams     []uint          `json:"teams"`
	Groups    []*models.Group `json:"groups"`
}

File: frontend/board/src/app/modules/kanban/models/kanban-user.ts
import { Group } from "./group";
import {KanbanIssue} from "./kanban-issue";

export type KanbanUser = {
  id: number
  name: string
  username: string
  avatarUrl: string
  issues: KanbanIssue[]
  teams: number[]
  groups: Group[]
}

export type KanbanUserResponse = {
  users: KanbanUser[],
  updateTime: string | null
}
